lib/content-services/src/lib/search/components/search-input/search-input.component.ts

```diff
@@ -45,6 +47,15 @@ export class SearchInputComponent {
     @Output()
     changed = new EventEmitter<string>();
 
+    constructor(private appConfig: AppConfigService) {}
+
+    ngOnInit(): void {
+        const searchConfig = this.appConfig.get<SearchConfiguration>('search-headers') || {};
```

Comment:
Also I think it should take it from `search` which is the default search config instead of `search-headers`

Reply:
done :)